bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/Enums.java

```diff
@@ -148,65 +148,63 @@ public String getValue() {
         OFF(""),
         UNKNOWN("??");
 
-        private static final Logger LOGGER = LoggerFactory.getLogger(SpecialMode.class);
+        private static final Logger logger = LoggerFactory.getLogger(AdvancedMode.class);
```

Comment:
As it is a static, the good practice us uppercase so LOGGER was good.

Reply:
> This is a breaking change to be explained in release notes.

Where is the "Release notes"? Is it something for the openhab maintainer (i.e. not me) to do?

Reply:
fixed